### PR TITLE
Fix *_nomg() functions and their usage

### PR DESCRIPTION
**Pull request by pali**

### CODE REVIEW

**Comment by choroba** [comment]
The failed tests don't seem to be relevant to the changes. Can you rerun them?

**Comment by pali** [comment]
Already pressed "re-run failed jobs" button.

File: dbdimp.h [hunk 1]
```diff
@@ -222,7 +222,7 @@ PERL_STATIC_INLINE char * SvPVbyte_nomg(pTHX_ SV *sv, STRLEN *len)
 #ifndef SvIV_nomg
 PERL_STATIC_INLINE IV SvIV_nomg(pTHX_ SV *sv)
 {
-  IV *iv;
+  IV iv;
```
Comment (choroba) [inline dbdimp.h:225]: Is the code tested in any of the GH Actions?

Comment (pali) [reply dbdimp.h:225]: Yes, this fixup commit is fixing warning (in commit message) which was discovered here by github actions.

**Review by choroba — APPROVED** [review]

**Comment by pali** [comment]
squashed and merged

**Pull request merged**